SMLHelper/Handlers/CraftDataHandler.cs

```diff
@@ -61,7 +61,7 @@ public static void EditHarvestOutputList(TechType techType, TechType harvestOutp
         /// </summary>
         /// <param name="techType">The TechType whose HarvestType you want to edit.</param>
         /// <param name="harvestType">The HarvestType for that TechType.</param>
-        public static void EditHarvestTypeList(TechType techType, HarvestType harvestType)
+        public static void EditHarvestType(TechType techType, HarvestType harvestType)
```

Comment:
This is a good change. The original method name is informed too much by the underlying implementation.
The new method name is more straight to the point about what it intends to accomplish.